types/AnimeList.ts

```diff
@@ -0,0 +1,97 @@
+export interface Image {
+  image_url: string;
+  small_image_url: string;
+  large_image_url: string;
+}
+
+export interface Reference {
+  mal_id: number;
+  type: string;
+  name: string;
+  url: string;
+}
+
+export interface Anime {
+  mal_id: number;
+  url: string;
+  images: {
+    jpg: Image;
+    webp: Image;
+  };
+  trailer: {
+    youtube_id: string;
+    url: string;
+    embed_url: string;
+  };
+  approved: true;
+  titles: string[];
+  title: string;
+  title_english: string;
+  title_japanese: string;
```

Comment:
I think it's not necessary to add to the interface all fields for this case because it will not potentially spread in the future. But maybe nice to have it in real projects if we potentially will use all fields